tasks/gcc.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * google-code-prettify
+ * https://github.com/google/code-prettify
+ *
+ * @author Amro
+ * @license Apache-2.0
+ */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  /*
+   * HACK: Custom task to override "closure-compiler:langs" since all lang-*
+   * files are compiled in parallel, launching too many Java processes at once
+   * which easily runs out of memory! This task programmatically creates
+   * separate targets (one file per target) so that lang files are compiled
+   * sequentially (sync) instead of in parallel (async).
+   */
+  grunt.registerMultiTask('gcc', 'Override closure-compiler', function () {
```

Comment:
It blows my mind that someone would use "gcc" as an abbreviation for Google Closure Compiler.